src/app/frontend/storageclass/detail/detail.html

```diff
@@ -16,4 +16,11 @@
 
 <div layout="column">
   <kd-storage-class-info storage-class="::$ctrl.storageClass"></kd-storage-class-info>
+  <kd-content-card>
+    <kd-content>
+      <kd-persistent-volume-card-list persistent-volume-list="::$ctrl.storageClass.persistentVolumeList"
+                                      persistent-volume-list-resource="::$ctrl.storageClassPersistentVolumesResource">
+      </kd-persistent-volume-card-list>
```

Comment:
The issue with sorting is most probably here. Take a look on replica set's pods or events. It should work.

Reply:
Fixed searching and sorting issue in persistentvolume list.